js/scripts.js

```diff
@@ -90,7 +90,7 @@ var MoviePoster = React.createClass({
 
   render: function () {
     return (
-      React.createElement('img'alt="poster", {src: this.props.moviePoster})
+      React.createElement('img'alt:"poster", src: this.props.moviePoster}  
```

Comment:
Łukasz, this is not working, in console you have a bug for sure ;)

you have to open and close your object - props are objects. Look, it starts with `{` and end with `}` - as object ;)

it should be: `React.createElement(TAG_NAME, PROPS_OBJECT);`
so `React.createElement('img', {});`

object with props can look that: 
```
const props = {
  src: this.props.moviePoster,
  alt: "poster"
}
```

so: `React.createElement('img', props);`